scipy/signal/_filter_design.py

```diff
@@ -1231,6 +1274,28 @@ def sos2tf(sos):
     Notes
     -----
     .. versionadded:: 0.16.0
+
+    Examples
+    --------
+    Find the polynomial representation of the transfer function H(s) 
+    using its 'sos' (second-order sections).
+
+    .. math::
+    
+        H(s) = \frac{ 0.5s + 1 }{ s^2 + 2s - 15 } * \frac{ s^2 - 7s - 2 }{ s^3 + s^2 -2 }
```

Comment:
Not entirely clear how this relates to the example

Reply:
Fixed it. it was a different example. thx for noticing.